bundles/framework/divmanazer/extension/ExtraFlyout.js

```diff
@@ -97,9 +99,13 @@ Oskari.clazz.define('Oskari.userinterface.extension.ExtraFlyout',
                 } else {
                     me.options.container.append(popup);
                 }
-                popup.find('.icon-close').on('click', function () {
+
+                var closeButton = popup.find('.oskari-flyouttool-close');
+                closeButton.empty().append(popup.find('.icon-close'));
```

Comment:
This row looks like it doesn't really do anything. It removes the icon div and appends it back? Is it needed for some reason or could it be removed? Maybe add a comment why it's there if it's needed? Otherwise it's will be removed by accident as it looks like it's dead code.